go/interpreter/lfvm/memory_test.go

```diff
@@ -47,3 +52,232 @@ func TestExpansionCosts(t *testing.T) {
 		}
 	}
 }
+
+func TestMemory_CopyWord(t *testing.T) {
+
+	testValue := []byte{0x12, 0x34, 0x56, 0x78, 0x90, 0xab, 0xcd, 0xef}
+
+	tests := map[string]struct {
+		gas            tosca.Gas
+		offset         uint64
+		expectedTarget uint256.Int
+		expectedError  error
+	}{
+		"regular": {
+			gas:            100,
+			offset:         0,
+			expectedTarget: *uint256.NewInt(0).SetBytes8(testValue),
+			expectedError:  nil},
+		"not enough gas": {
+			gas:            6,
+			offset:         64,
+			expectedTarget: *uint256.NewInt(0),
+			expectedError:  errOutOfGas},
+	}
+
+	for name, test := range tests {
+		t.Run(name, func(t *testing.T) {
+			ctxt := getEmptyContext()
+			m := NewMemory()
+			target := uint256.NewInt(0)
+			m.store = make([]byte, 32)
```

Comment:
Initialize data to other than zero: By having zero values, you cannot differentiate zero initialization from faulty assignment.